### DISCUSSION TITLE
Some issues with the Mono theme

### DISCUSSION THREAD

**Discussion by Backspaze** [GetPublii/Publii#1616] [Help]
I recently started using Publii to create a new web site and decided to try [the Mono theme](https://marketplace.getpublii.com/themes/mono/), but I ran into some issues and I  thought I'd create a discussion here to see if someone might be able to help.

**Edit the text for the newsletter subscription**

I found #1432 regarding this. Are there any plans to make customizing this text accessible directly in Publii?

**Remove post count from all tag views**

Under **Theme** > **Custom settings** > **Tag options** there's a setting called **Display post counter**. The description above the settings says that these settings will affect all tags on the site, but that doesn't seem to be the case. If you disable the post counter and go to the tag list at yoursite.com/tags/, the post counter is still shown on the tags in that view.

Could this be considered a bug for the theme, or is it by design, and to fix it you need customize the theme?

**Option to remove duplicate website logo**

In the screenshot at [the site for the theme](https://marketplace.getpublii.com/themes/mono/), there's only one website logo in the top left corner (I've embedded the screenshot below). But if you create a site with the theme, you get one logo at the top and one at the bottom. The top logo works as a home button, but the bottom one just points to the same page you're already on. What is the purpose of this and can we remove the bottom one without customizing the theme?

![bild](https://github.com/user-attachments/assets/5ce6dee8-ae48-4ced-a8be-c6a05cf0af7d)

**Customize alignment of follow buttons in sidebar**

When you have one or three or more sharing buttons, it looks good, but stick with two and it doesn't look as nice. The two buttons are placed left-aligned and right-aligned with a big space between. I'm guessing the only way to fix this is to customize the theme?

**One**
![bild](https://github.com/user-attachments/assets/5e31d238-cda0-41d5-9e59-f1b15000d5ae)

**Two**
![bild](https://github.com/user-attachments/assets/ee2dc650-7329-41d1-addf-94148ba80dbf)

**Three**
![bild](https://github.com/user-attachments/assets/50acedbc-4a48-4f23-80e0-6e63015aae63)

**Four**
![bild](https://github.com/user-attachments/assets/8b910ac4-c383-4eb4-a8a1-0d5ecc7cf759)

**Search**

Are there any plans to add search support for this theme?

**Comment by bobmitro** [comment]
Hey,

The theme's text strings can be found in the [language file](https://getpublii.com/docs/translate-publii-theme-to-another-language.html), which you can manually edit.  We plan to release a plugin to simplify editing these texts, I think, this year.

As the name suggests, the post counter option originally only applied to individual tag pages where the number of posts under a specific tag is displayed. However, I have now added an option that extends this functionality to the tag counter on the /tags/ page, where all tags are listed. 

Mono version 2.0.1.0 has resolved the issue with the bottom logo. Now, both logos will correctly link to the homepage. 

These updates have been implemented in Mono version [2.0.1.0](https://marketplace.getpublii.com/themes/mono/).

The Follow icons are stretched in the original theme by Huseyin Emanet, so this aspect has been retained without changes in the Publii theme version. However, if you'd like to adjust the alignment to avoid stretching, you can easily do so using the Custom CSS Tool by adding the following code: `.follow {
    justify-content: start;
}
`

The search is not planned.

**Reply by Backspaze** [reply]
Thanks for the reply and updating the theme!

A plugin for editing those texts sounds great, and preferable to editing the language files. Looking forward to it.

I've tried the updated theme and can confirm that the post count is gone at /tags/ for me as well.

Regarding the logo, my main issue was why there was two logos to begin with. I think it looks much better with just one, as in the screenshot. I've removed the bottom logo from my site by adding the custom CSS shown below. I chose `visibility: hidden` instead of `display: none` as the latter messes up the position of the menu.

```css
.logo--atbottom {
    visibility: hidden;    
}
```

I've also played around with custom CSS for `.follow` per your suggestion and got it working the way I prefer.

I first tried to create a custom version of the Mono theme by downloading it, copying the original downloaded files and then editing the **config.json** file to update the name and version as shown in the excerpt below.

```json
"name": "MonoCustom",
"version": "1.0.0.0",
```

I then tried to make the changes I wanted in my custom theme. Since I've never worked with a Publii theme before, I thought I'd start small, and the only change I made was Edit the text `Subscribe to Mono` to `Subscribe to Lab` in **mono.lang.json**. For some reason that did not work, and the only thing that happened was the the corresponding text on the website was shown as `[MISSING TRANSLATION]`.

After digging some more, I chose to instead use the method described in the [translation guide](https://getpublii.com/docs/translate-publii-theme-to-another-language.html) you linked to. And for the rest of my changes I either used the custom CSS tool or [theme overrides](https://getpublii.com/dev/theme-overrides/) which worked much better.

I have no clue why my custom theme didn't work. I only changed the name and version in the **config.json** file so that the theme would be easily distinguished in Publii. But for some reason, just changing one word in **mono.lang.json** caused the theme to show the text about missing translation. At least I was able to work around it by using the other methods described above.

**Reply by bobmitro** [reply]
If you change the theme name, you should also change the language file to **YourThemeName**.lang.json.

**Reply by Backspaze** [reply]
Ah! Thanks for clearing that up. Now that I know that, I almost want to give it another shot at customizing the theme, but using the override options seems like the better way in the long run, so I think I'll stick with that.
